libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -1561,6 +1568,53 @@ export const buildTigerSpecificFunctions = (
         });
     },
 
+    exportRawData: async (execution, projectId, fileName) => {
```

Comment:
I'm wondering whether we need this in tiger specific functions, when we have https://github.com/gooddata/gooddata-ui-sdk/pull/5557/files#diff-79080bcabf02f4997926c1c0771b1859b77386063ce55e3fa49fc68fcc6b8b3fR289. 
I don't see it used, but I may be missing something.

Reply:
Good point, removed the specific endpoint call from `tigerSpecificFunctions` and use global introduced as part of execution result.